src/test/java/org/example/FilteringSearching.java

```diff
@@ -1,5 +1,147 @@
 package org.example;
 
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Random;
+
 public class FilteringSearching {
```

Comment:
Best practice is that classes that contain @Tests should have the suffix to reflect that -> FilteringSearchingTests

Reply:
Changed class name.